core/src/main/java/org/jdbi/v3/core/transaction/TransactionHandler.java

```diff
@@ -28,55 +28,55 @@
 public interface TransactionHandler
 {
     /**
-     * Called when a transaction is started
+     * Called when a transaction is started.
```

Comment:
The wording on this and other methods in the class seems to imply that the handler is only being _informed_ about the change on the transaction, when in reality the handler is _responsible_ for carrying out that change.

I think something like `Begins a transaction on the given handle` would be more accurate.